src/store.js

```diff
@@ -82,6 +82,11 @@ const rootSagas = function* () {
   ]);
 };
 
+const logger = createLogger({
+  predicate: (getState, action) => !['SYSTEM_POLL_START', 'SYSTEM_POLL_SUCCESS'].includes(action.type),
+  collapsed: true,
+});
+
```

Comment:
@rhenshaw56 what does this do?  Suppress system pool messages?  Probably a good idea, they were starting to clog my console!